standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/properties/HMSServletTest.java

```diff
@@ -63,21 +67,21 @@
 
 public class HMSServletTest extends HMSTestBase {
```

Comment:
There are some test classes for the same purpose, i.e, the HiveServletTest1A, HiveServletTestA, HiveServletTest1, HiveServletTest, maybe we can just keep only one of them.

Reply:
They do use different server/client implementations (direct/thrift/servlet + URL/HttpClient) to perform the same calls.